test/data/Vanilla/Bukkit/Paper/paper-plugin-unsupported-class-version-1-16-5.json

```diff
@@ -464,7 +464,7 @@
     "analysis": {
         "problems": [
             {
-                "message": "The plugin 'Slimefun4-1027' requires Java version 16 or later to run.",
+                "message": "The plugin 'Slimefun4-1027' requires Java version -44 or later to run.",
```

Comment:
Looks like the class version wasn't read correctly

Reply:
Fixed